drivers/include/at86rf2xx.h

```diff
@@ -89,14 +90,15 @@ extern "C" {
  * @brief   Flags for device internal states (see datasheet)
  * @{
  */
-#define AT86RF2XX_STATE_TRX_OFF      (0x08)     /**< idle */
-#define AT86RF2XX_STATE_PLL_ON       (0x09)     /**< ready to transmit */
-#define AT86RF2XX_STATE_SLEEP        (0x0f)     /**< sleep mode */
-#define AT86RF2XX_STATE_BUSY_RX_AACK (0x11)     /**< busy receiving data */
-#define AT86RF2XX_STATE_BUSY_TX_ARET (0x12)     /**< busy transmitting data */
-#define AT86RF2XX_STATE_RX_AACK_ON   (0x16)     /**< wait for incoming data */
-#define AT86RF2XX_STATE_TX_ARET_ON   (0x19)     /**< ready for sending data */
-#define AT86RF2XX_STATE_IN_PROGRESS  (0x1f)     /**< ongoing state conversion */
+#define AT86RF2XX_STATE_FORCE_TRX_OFF  (0x03)     /**< force transition to idle */
```

Comment:
This is only a command and not a state, so `_set_state` should be changed to something like this:

    #define _set_state(dev, state)        _set_state_ex(dev, state, state)

	static inline void _set_state_ex(at86rf2xx_t *dev, uint8_t cmd, uint8_t state)
	{
		at86rf2xx_reg_write(dev, AT86RF2XX_REG__TRX_STATE, cmd);
		while (at86rf2xx_get_status(dev) != state);
		dev->state = state;
	}